src/test/components/__snapshots__/StackChart.test.js.snap

```diff
@@ -265,6 +265,31 @@ Array [
     286.3333333333333,
     43,
   ],
+  Array [
+    "set fillStyle",
+    "#ffe90070",
+  ],
+  Array [
+    "fillRect",
+    350,
+    48,
+    66.4,
+    15,
+  ],
+  Array [
+    "measureText",
+    "componentC",
+  ],
+  Array [
+    "set fillStyle",
+    "#000000",
+  ],
+  Array [
+    "fillText",
+    "componentC",
+    353,
+    59,
+  ],
```

Comment:
do you know why we have _more_ boxes getting drawn in the test? From your comment I was expecting either as many or less boxes.

Reply:
This is from the marginRight fix - we weren't drawing boxes in the right margin before. This test sets up a drawing area that's 150px marginLeft + 200px GRAPH_BASE_WIDTH + 15px marginRight, so the right margin starts at 350px, and the new box is also drawn at 350.